src/transformStateWithClones.js

```diff
@@ -8,6 +8,49 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
```

Comment:
I suggest deleting unnecessary comments just for the sake of neatness of the code)

Reply:
Remove comment